target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -247,6 +247,18 @@ define Device/redmi_ax6
 endef
 TARGET_DEVICES += redmi_ax6
 
+define Device/spectrum_sax1v1k
+	$(call Device/FitImage)
+	DEVICE_VENDOR := Spectrum
+	DEVICE_MODEL := SAX1V1K
+	DEVICE_DTS_CONFIG := config@rt5010w-d187-rev6
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	SOC := ipq8072
+	DEVICE_PACKAGES := ipq-wifi-askey_sax1v1k e2fsprogs kmod-fs-ext4 losetup
```

Comment:
`e2fsprogs kmod-fs-ext4 losetup` you don't need this, it's done by default

Reply:
fixed and confirmed it compiles and works fine